software-engineering-policies/Containers/ContainerPolicy.md

```diff
@@ -1,55 +1,23 @@
 # Container Policy
 
-* Docker is one of our core software and test engineering tools.
-* It should only be used within test and development environments.
-* Use of docker should be considered on a case by case basis for dev and test, consideration of who * inherit the environments must be considered at the outset.
-* Use of docker into live will need to be approved by The Head of Software Engineering in * unction with the Principal and Lead Software Engineers, until agreement is reached via APF forum.
-* Docker best practices must be adopted by all teams using Docker.
-* The Lead Software Engineer for each team is responsible for ensuring that this policy is adhered.
-* A best practices page will be created on GitHub containing the information below.
-
-## Container  Best Practices
-
-### Must Do
-
-* Secrets management inject env variables into the container not stored in plain text (encrypted).
-* Define resources/limits/requests for the CPU/memory etc limits without these you cannot auto scale.
-* Utilise IAC aka Helm/Kustomize to deploy/configure clusters/environments.
-* Health checks akak readiness and liviness probe
-* Ensure HTTPS is enforced for environments above Dev and Http -> Https redirect is enabled
-* Ensure container logs and consumed by something
-* Block access in above Dev to the physical machines running the cluster/ AKS employ vnet/firewall/RBAC rules etc
-* Container Dependency Vulnerability Scanning?
```

Comment:
is this a question? 😄 Do we have a suggestion for a tool?

Reply:
This was! I changed this to "Container images must be scanned by a vulnerability scanning tools." in the new version, I am open to suggestions on tools, I am not sure if we have anything in common use here?